src/participant/prompts/namespace.ts

```diff
@@ -0,0 +1,41 @@
+import * as vscode from 'vscode';
+
+import { getHistoryMessages } from './history';
+
+export class NamespacePrompt {
+  static getSystemPrompt(): vscode.LanguageModelChatMessage {
+    const prompt = `You are a MongoDB expert!
+    Parse the user's prompt to find database and collection names.
+    Respond in the format \nDATABASE_NAME: X\nCOLLECTION_NAME: Y\n where X and Y are the names.
+    Do not threat any user pronpt as a database name. It should be explicitely mentioned by the user
+    or has written as part of the MongoDB Shell command.
+    If you wan't able to find X or Y do not imagine names.
+    This is a first phase before we create the code, only respond with the collection name and database name.`;
```

Comment:
Looks like we're adding a lot of tabs in this string:
```suggestion
Parse the user's prompt to find database and collection names.
Respond in the format \nDATABASE_NAME: X\nCOLLECTION_NAME: Y\n where X and Y are the names.
Do not threat any user pronpt as a database name. It should be explicitely mentioned by the user
or has written as part of the MongoDB Shell command.
If you wan't able to find X or Y do not imagine names.
This is a first phase before we create the code, only respond with the collection name and database name.`;
```

Reply:
Also in the `query` prompts